build/react-gallery-swiper.js

```diff
@@ -187,7 +187,7 @@ return /******/ (function(modules) { // webpackBootstrap
 	            var newImages = nextProps.images,
 	                newLazyload = nextProps.lazyLoad;
 
-	            if (images !== newImages) {
+	            if (images !== newImages || images.length !== newImages.length) {
```

Comment:
I didn't add that code, it is on the compiled code. My guess is that the previous version did not update the compiled code and this is something present in previous changes.